doc/configuration-v3_2_experimental.md

```diff
@@ -131,6 +131,10 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_options_** (list of strings): any additional options to be passed to the cryptsetup utility.
     * **_wipeVolume_** (boolean): whether or not to wipe the device before volume creation, see [the documentation on filesystems](operator-notes.md#filesystem-reuse-semantics) for more information.
     * **_clevis_** (object): describes the clevis configuration for the luks device.
+      * **_custom_** (object): override the clevis configuration. The `pin` & `config` will be passed directly to `clevis luks bind`. If specified all other clevis options must be excluded.
+        * **config** (string): the clevis configuration JSON.
+        * **_netdev_** (bool): whether or not the device requires networking.
```

Comment:
Is `netdev` the clearest name for the user-facing field?  The `_netdev` mount option is somewhat obscure, so we don't necessarily need to be consistent with it.  Maybe something like `needsNetwork`?